boards/arm/olimex_stm32_p405/doc/olimex_stm32_p405.rst

```diff
@@ -0,0 +1,270 @@
+.. _olimex_stm32_p405:
+
+OLIMEX-STM32-P405
+#################
+
+Overview
+********
+
+Zephyr applications use the olimex_stm32_p405 board configuration
+to run on the OLIMEX-STM32-P405 open source hardware. It is based on
+the STMicroelectronics STM32F405RG ARM Cortex-M4 CPU.
+
+.. figure:: img/olimex-stm32-p405.png
+     :width: 540px
+     :align: center
+     :height: 454px
+     :alt: OLIMEX-STM32-P405
+
+     OLIMEX-STM32-P405
+
+More information about the board can be found at the
```

Comment:
Following the template, let's add the hardware section title before this paragraph:

    Hardware
    ********

    Information about the board can be found at the